libs/features/webapp-prompt/webapp-prompt.js

```diff
@@ -212,7 +272,7 @@ class AppPrompt {
   };
 
   initRedirect = () => setTimeout(() => {
-    this.close({ saveDismissal: false });
+    this.close({ saveDismissal: false, dismissalActions: false });
```

Comment:
Could you test this by defining a hash as the redirect URL? That would avoid the page reload, but would trigger the UI changes.

Reply:
Unfortunately this still results in this line ostensibly never being run. It's difficult to say why, but it may have something to do with how fakeTimers interacts with the event loop.n